### ISSUE TITLE
ZeresPluginLibrary Error

### ISSUE THREAD

**Issue by aloopakoda** [JustOptimize/return-ShowHiddenChannels#141]
ZeresPluginLibrary is not loading. Please help.

**Comment by yu06100** [comment]
I am also having this same

**Comment by JustOptimize** [comment]
I'm not he author of zerespluginlibrary, if the library stopped working Zeres will fix it

**Comment by angrybirdice** [comment]
> I'm not he author of zerespluginlibrary, if the library stopped working Zeres will fix it

I do agree with you, I was unaware of the situation with these plugins but I understand.

**Comment by angrybirdice** [comment]
for zerepluginlibrary 
just go to https://github.com/rauenzi/BDPluginLibrary and look its issue
disclaimer before something happens just in case I want to leave it out, so I don't be held responsible and stressed about it that I posted this, also I'm not the author of zerespluginlibrary.

I don't completely recommend harassing, complaining, etc (if it happens).
I already started to understand the situation every day for me it down longer than I thought it would
take your time if you an author of a plugin.
